src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -61,45 +61,60 @@ class ItemRecognition extends Component {
     );
   };
 
+  setSuggestions = (items, index) => {
+    const suggestions = [];
+    while (suggestions.length < 3) {
+      if (items[index].id === 'unknown') continue;
```

Comment:
We can use the built in array methods to avoid recreating the wheel a bit,

Something like...

```
const suggestions = items
  .slice(index)
  .filter(item => item.id !== 'unknown')
  .slice(0, 3)
  .map(item => item.id)
```

Lodash has some handy methods to go a step further and use lazy methods so we're not filtering an entire array to just then take the first 3 items, e.g.

```
const suggestions = _.chain(items)
  .slice(index)
  .filter(item => item.id !== 'unknown')
  .take(3) // or .slice(0, 3)
  .value();
```